tests/clients/github/test_github.py

```diff
@@ -30,10 +30,17 @@ def tearDown(self):
             j.sals.fs.rmtree(f"/tmp/{self.directory_name}/")
 
     def wait_for_deleting_repo(self, sec, repo):
-        while sec:
-            if repo.full_name in str(self.client.get_repos()):
+        for i in range(sec):
```

Comment:
In case of we don't need the counter variable, it's better to use `_` instead.
```suggestion
        for _ in range(sec):
```